database/schema.sql

```diff
@@ -5,7 +5,7 @@ CREATE DATABASE fetsy;
 \c fetsy;
 
 CREATE TABLE IF NOT EXISTS items (
-  item_id INT PRIMARY KEY,
+  item_id SERIAL PRIMARY KEY,
   item_name VARCHAR(150),
   best_seller BOOLEAN,
   price float(2),
```

Comment:
maybe not a big deal but i've read to use inte for price rather than float https://stackoverflow.com/questions/15726535/postgresql-which-datatype-should-be-used-for-currency

Reply:
Did you mean numeric(2) ?